content/applications/hr/payroll/work_entries.rst

```diff
@@ -80,95 +93,186 @@ there are any errors on the work entry, such as required fields being left blank
 
 .. note::
-   Payslips will have a status of *Waiting* until the :guilabel:`Create Draft Entry` button has been
+   Payslips will have a status of *Waiting* until the :guilabel:`Create Draft Entry` button is
    clicked. After, the payslip status will change to *Done*.
 
-Payslips can be printed by clicking the box next to each payslip to print, or clicking the box next
-to :guilabel:`Reference` to select all the payslips at once. Click the :guilabel:`Print` button, and
-a PDF file will be created with all the specified payslips.
+Printing payslips
+-----------------
+
+To print payslips, first view the individual payslips by clicking the :guilabel:`Payslips` smart
+button on the batch form. Next, select the payslips to print in the payslip list. Click the box next
+to each payslip to print, or click the box next to :guilabel:`Reference` to select all the payslips
+in the list at once. Click the :guilabel:`Print` button, and a PDF file is created with all the
+specified payslips.
 
 .. image:: work_entries/print-payslips.png
    :align: center
    :alt: Print button for printing the payslips.
 
+.. note::
+   The :guilabel:`Print` button does not appear until at least one payslip is selected in the list.
+
+Time Off to Report
+==================
+
+If a time-off request is submitted for a time period that was already processed on a payslip, the
+time-off request appears in the :guilabel:`Time Off to Report`, with a status of :guilabel:`To defer
+to next payslip`. This is because the employee was already paid for that day, and it was logged as
+time spent at work, as a typical work day.
+
+In order to keep the employee's time-off balances correct, the time-off request must be applied to
+the following pay period. This not only ensures time-off request balances are current, but this also
+eliminates the need to redo work entries, cancel paychecks, and reissue paychecks.
+
+The most common scenario when this situation occurs, is when payslips are processed a day or two
+before the pay period ends, and an employee is unexpectedly sick on one of the last days of the pay
+period. The employee puts in a time-off request for a day that was already processed on a payslip as
+a regular work day. Instead of canceling the payslip, modifying the work entries, and reissuing the
+paycheck, Odoo allows for those time-off requests to be applied to the following pay period instead.
+
+To view all the time-off requests that need to be deferred to the next payslip, navigate to
+:menuselection:`Payroll app --> Work Entries --> Time Off to Report`. The default filter for this
+report is :guilabel:`To Defer`.
+
+All time-off requests that need to be applied to the following pay period appear, with a
+:guilabel:`Payslip State` of :guilabel:`To defer to next payslip`.
+
+.. image:: work_entries/time-off-to-report.png
+   :align: center
+   :alt: A list of all time-off requests that were not approved before payslips were generated.
+
+Defer multiple time-off requests
```

Comment:
```suggestion
Defer multiple time off requests
```

Reply:
Should this heading be "defer multiple work entries"? Time off requests are not mentioned in this section, only work entries.

Reply:
I am torn because the only work entries that can be deferred are ones related to time off (like vacation, sick time, etc). So even though they are work entries, it is only for specific work entries that are for a time off type. A regular work entry would not appear here. In the settings section, the option to "Defer Time Off" appears in the "Time Off" section since it only applies to time off entries. SO I thought to be very clear, I would say time off in this heading so users know that it's only for time off entries.